pyramid_chameleon/precompile.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
```

Comment:
This is not required for scripts installed using setuptools. I would also recommend against hardcoding the path, especially since that is not likely the path for anyone using Python 3 or alternate python versions.